### PR TITLE
PEPPPER-298 Incremental CircleCI config refactor

### PR DESCRIPTION
**Pull request by aweng98**
Part 2 refactoring to reduce code clutter.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -591,77 +592,11 @@ jobs:
...
-            export emailWaitTime=$(vault read --format=json secret/pepper/test/v1/e2e | jq -r ".data.email.minWaitTime")
-            echo "export EMAIL_REFRESH_TOKEN=$refreshToken" >> playwright-env/envvars
-            echo "export EMAIL_CLIENT_ID=$emailClientId" >> playwright-env/envvars
-            echo "export EMAIL_CLIENT_SECRET=$emailClientSecret" >> playwright-env/envvars
-            echo "export EMAIL_REDIRECT_URL=$emailRedirectUri" >> playwright-env/envvars
-            echo "export MIN_EMAIL_WAIT_TIME=$emailWaitTime" >> playwright-env/envvars
-
+            pwd
+            set -u
+            .circleci/export-playwright-env.sh
```
Comment (aweng98) [inline .circleci/config.yml:596]: New shell script to reduce code clutter in config.ym. No change in contents.

File: .circleci/config.yml [hunk 2]
```diff
@@ -575,7 +576,7 @@ jobs:
       - run-linter-check
       - run-tests
 
-  build-playwright-test-job:
+  playwright-build:
```
Comment (aweng98) [inline .circleci/config.yml:579]: Simplfiy job name. And I'm setting up new job naming convention: begin with a noun instead a verb.

File: .circleci/config.yml [hunk 3]
```diff
@@ -465,9 +465,10 @@ commands:
             or:
               - equal: [ UNKNOWN, << parameters.env >> ]
               - equal: [ prod, << parameters.env >> ]
+              - equal: [ staging, << parameters.env >> ]
```
Comment (aweng98) [inline .circleci/config.yml:468]: Playwright tests are not supported on Staging at this time.

File: .circleci/config.yml [hunk 4]
```diff
@@ -681,7 +616,7 @@ jobs:
             - playwright-env # dir which contains env-variables file
             - playwright-e2e
 
-  run-playwright-test-job:
+  playwright-test:
```
Comment (aweng98) [inline .circleci/config.yml:616]: Simplfiy job name.

**Review by Rinchan20 — APPROVED** [review]

**Pull request merged**